math/src/field/extensions/cubic.rs

```diff
@@ -127,6 +127,10 @@ where
     fn from_base_type(x: [FieldElement<Q::BaseField>; 3]) -> [FieldElement<Q::BaseField>; 3] {
         x
     }
+
+    fn representative(_x: Self::BaseType) -> Self::BaseType {
+        todo!()
```

Comment:
[Same](https://github.com/lambdaclass/lambdaworks/pull/59/files#r1123112236) for this

Reply:
https://github.com/lambdaclass/lambdaworks/issues/139